packages/xrpl/test/integration/transactions/escrowFinish.test.ts

```diff
@@ -1,71 +1,97 @@
 import { assert } from 'chai'
-import _ from 'lodash'
 import { EscrowFinish, EscrowCreate } from 'xrpl-local'
 
 import serverUrl from '../serverUrl'
-import { setupClient, teardownClient } from '../setup'
-import { generateFundedWallet, getXRPBalance, testTransaction } from '../utils'
+import {
+  setupClient,
+  teardownClient,
+  type XrplIntegrationTestContext,
+} from '../setup'
+import {
+  calculateWaitTimeForTransaction,
+  generateFundedWallet,
+  getXRPBalance,
+  testTransaction,
+} from '../utils'
 
 // how long before each test case times out
-const TIMEOUT = 20000
+const TIMEOUT = 30000
 
 describe('EscrowFinish', function () {
-  this.timeout(TIMEOUT)
+  let testContext: XrplIntegrationTestContext
 
-  beforeEach(_.partial(setupClient, serverUrl))
-  afterEach(teardownClient)
+  beforeEach(async () => {
+    testContext = await setupClient(serverUrl)
+  })
+  afterEach(async () => teardownClient(testContext))
 
-  it('base', async function () {
-    // get the most recent close_time from the standalone container for cancel & finish after.
-    const CLOSE_TIME: number = (
-      await this.client.request({
-        command: 'ledger',
-        ledger_index: 'validated',
-      })
-    ).result.ledger.close_time
-    const wallet1 = await generateFundedWallet(this.client)
+  it(
+    'base',
+    async () => {
+      const wallet1 = await generateFundedWallet(testContext.client)
 
-    const AMOUNT = 10000
+      // get the most recent close_time from the standalone container for cancel & finish after.
+      const CLOSE_TIME: number = (
+        await testContext.client.request({
+          command: 'ledger',
+          ledger_index: 'validated',
+        })
+      ).result.ledger.close_time
 
-    const createTx: EscrowCreate = {
-      Account: this.wallet.classicAddress,
-      TransactionType: 'EscrowCreate',
-      Amount: '10000',
-      Destination: wallet1.classicAddress,
-      FinishAfter: CLOSE_TIME + 2,
-    }
+      const waitTimeInMs = calculateWaitTimeForTransaction(CLOSE_TIME)
 
-    await testTransaction(this.client, createTx, this.wallet)
+      const AMOUNT = 10000
 
-    const initialBalance = await getXRPBalance(this.client, wallet1)
+      const createTx: EscrowCreate = {
+        Account: testContext.wallet.classicAddress,
+        TransactionType: 'EscrowCreate',
+        Amount: '10000',
```

Comment:
nit: It seems like this field should be using the `Amount` variable above (converted to a string)

Reply:
Fixed